File: src/main/java/com/ptah/entity/userprofiling/OrganizationNomination.java
package com.ptah.entity.userprofiling;

import com.ptah.common.BaseEntity;
import jakarta.persistence.Entity;
import jakarta.persistence.EnumType;
import jakarta.persistence.Enumerated;
import jakarta.persistence.ManyToOne;
import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
public class OrganizationNomination extends BaseEntity {
    @ManyToOne
    private User user;
    @ManyToOne
    private Organization organization;
    private String organizationRole;
}

File: src/main/kotlin/com/ptah/entity/userprofiling/OrganizationNomination.kt
package com.ptah.entity.userprofiling

import com.ptah.common.BaseEntity
import jakarta.persistence.Entity
import jakarta.persistence.ManyToOne
import lombok.Builder
import lombok.Getter
import lombok.Setter

@Entity
@Getter
@Setter
@Builder
class OrganizationNomination : BaseEntity() {
    @ManyToOne
    private val user: User? = null

    @ManyToOne
    private val organization: Organization? = null
    private val organizationRole: String? = null
}
